presto-testng-services/src/main/java/io/prestosql/testng/services/ReportUnannotatedMethods.java

```diff
@@ -140,6 +140,6 @@ private static boolean overrides(Method first, Method second)
 
     public static boolean isTemptoClass(Class<?> aClass)
     {
-        return "io.prestosql.tempto".equals(aClass.getPackage().getName());
+        return aClass.getPackage().getName().startsWith("io.prestosql.tempto");
```

Comment:
BTW Maybe we can deprecate & remove this interfaces?

Also, this should not match eg `io.prestosql.temptomat` -- you need to prefix match with a dot at the end.